app/assets/src/components/layout/narrow_container.scss

```diff
@@ -1,7 +1,7 @@
 .narrowContainer {
-  max-width: 1280px;
+  max-width: 1308px;
```

Comment:
please add a comment to explain how this value was chosen

Reply:
done